db-client-java/src/main/java/com/eventstore/dbclient/ConnectionSettingsBuilder.java

```diff
@@ -78,4 +81,14 @@ public ConnectionSettingsBuilder addHost(Endpoint host) {
         this._hosts.push(host);
         return this;
     }
+
+    public ConnectionSettingsBuilder keepAliveTimeout(long value) {
```

Comment:
**SUGGESTION:** Could we kee the naming consistent? In the builder above (https://github.com/EventStore/EventStoreDB-Client-Java/pull/60/files#diff-38d3d73124a50028533ed8eb19d8f761973f1aeafee7044034532e741fcf9ea9R32), we're using `keepAliveTimeoutInMs` while here we don't have `Ms` suffix. As other time-based settings are always provided with milliseconds, I'd suggest removing the `Ms` suffix.

Reply:
Done